diff.py

```diff
@@ -2578,8 +2588,26 @@ def process(dump: str, config: Config) -> List[Line]:
 
         if not row:
             continue
+        
+        # Check if the currrent line has "OFFSET <SYMBOL>:"
+        function_label_match = re.match(r"^[0-9a-f]+ <(.*)>:$", row)
+
+        if function_label_match:
+            function_name = function_label_match.groups()[0] + ":"
 
-        if re.match(r"^[0-9a-f]+ <.*>:$", row):
+            if config.diff_function_symbols:
+                # If diffing function symbols is enabled
+                # Add the symbol to the diff output
+                
+                output.append(
+                    Line(
+                        mnemonic=function_name, # this is needed for score changes
```

Comment:
This is the value that gets used for lining things up. In a case like
```
func_abc:
ins1
ins2
ins3
```
vs
```
func_xyz:
ins2
ins3
```
we want `func_abc:` and `func_xyz:` to line up, instead of `func_abc:` appearing on its own and `func_xyz` lining up with `ins1`, which would be an equally good scoring diff outcome if mnemonics don't match. So I would suggest putting a constant `mnemonic="<label>"` to counteract this.

`scorable_line` might need to be `"label " + function_name` or something like that for scoring to work in that case.